.golangci.yml

```diff
@@ -1,4 +1,13 @@
 linters-settings:
+  revive:
+    rules:
+      # https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#exported
+      - name: exported
+        severity: warning
+        disabled: false
+        exclude: [""]
+        arguments:
+          - "disableStutteringCheck"
```

Comment:
I disabled this because linter was throwing error for using `server.ServerConfig`

Reply:
we can name it as `server.Config`

Reply:
done

Reply:
but I have to admit that this is an annoying lint rule. Because it even throws the same error for private variables! They wont be accessed externally